test_scripts/API/Navigation/SendLocation/015_SendLocation_latitudeDegrees_bound_values.lua

```diff
@@ -0,0 +1,75 @@
+---------------------------------------------------------------------------------------------
+-- Requirements: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/TRS/embedded_navi/SendLocation_TRS.md
+--
+-- Requirement summary:
+-- In case mobile application sends valid SendLocation request, SDL must:
+-- 1) transfer Navigation.SendLocation to HMI
+-- 2) on getting Navigation.SendLocation ("SUCCESS") response from HMI, respond with (resultCode: SUCCESS, success:true)
+-- to mobile application.
+--
+-- Description:
+-- App sends SendLocation request with latitudeDegrees bound values.
```

Comment:
@GetmanetsIrina , please take a look at script 009 in SendLocation folder:
Seems that boundary values are checked already.

Also, this is a common comment as for atomic scripts.
I suggest to do not cut boundary checks by parameters.
Atomic test assumes there is only one test scenario verified. 

It is a very good to have separate scripts even by parameters, but as for me this is an overhead.
Each script = independent test scenario, think - it's enough.
Loops are not forbidden to iterate through params for the same test.

Reply:
@Itileda in 009 SendLocation script boundary values are checked in all scope of parameters.
In my opinion test is very common, in case of defect with some parameter script result will be not fully informative.
Agree with joining of tests with bound values of all parameters in one script.